src/schemas/block.yaml

```diff
@@ -20,7 +20,6 @@ Block:
     - size
     - transactions
     - uncles
-  additionalProperties: false
```

Comment:
Should this no longer be strict? Isn't block always fully specified, without optional additional fields?